src/high-level/high-level.lisp

```diff
@@ -930,6 +930,30 @@ with upper left block with dimension P-by-Q. Returns the intermediate representa
         (magicl.lapack-cffi::%zgeev jobvl jobvr rows a rows w vl 1 vr rows work lwork rwork info)
         (values (vector-to-list w) (make-matrix :rows rows :cols cols :data vr))))))
 
+(defun hermitian-eig (m)
+  "Finds the (right) eigenvectors and corresponding eigenvalues of a Hermitian square matrix M. Returns as two lists (EIGENVALUES, EIGENVECTORS) where the eigenvalues and eigenvectors are in corresponding order."
```

Comment:
Appears to return a list and a magicl:matrix, not two lists?

Reply:
Should be fixed in the documentation string of `eig` as well.